chef/cookbooks/crowbar/recipes/default.rb

```diff
@@ -390,7 +390,7 @@
     realm: realm
   )
 
-  notifies :reload, resources(service: "apache2")
+  notifies :restart, resources(service: "apache2"), :immediately
```

Comment:
Do we need the `:immediately`?

Also wondering how this would impact things when applying the crowbar proposal later on (if it ever rewrites that config file). Restarting apache means killing the connection that is used for the "apply" process, and I'm not sure it's very good.

What about just including the `mod_*` recipes before the `apache2` one?